main/webapp/modules/core/scripts/views/data-table/cell-ui.js

```diff
@@ -604,6 +604,13 @@ DataTableCellUI.prototype._startEdit = function(elmt) {
   cellDataType = (this._cell !== null && "t" in this._cell && this._cell.t !=  null) ? this._cell.t : cellDataType;
   elmts.typeSelect.val(cellDataType);
 
+  if (cellDataType === "date") {
+    var simpleDateFormatRef = $('<a></a>').attr('href', 'https://docs.oracle.com/javase/8/docs/api/java/text/SimpleDateFormat.html').append('here');
```

Comment:
There are a few problems here:
* the `here` word will not be translatable
* the position of the `here` word with the link is also not configurable by the translators (there could be languages where it does not make sense to have the link at the end of the text
* if we are to use this link, I would rather formulate like this "Supply dates based on the [SimpleDateFormat syntax](https://docs.oracle.com/javase/8/docs/api/java/text/SimpleDateFormat.html)" (shorter, more effective, avoids [the problems of "click here" links](https://heyoka.medium.com/dont-use-click-here-f32f445d1021))

Reply:
Oh, this is true. Thanks! 😃